### DISCUSSION TITLE
Why kraken

### DISCUSSION THREAD

**Discussion by 0-Captain** [openkraken/kraken#257] [General]
#### 为什么选择Kraken，项目README中解释：
- Quick development
- Cross platform
- Fast performance

但这些只是Kraken的优势，还不足以回答上述的问题。能否加一些使用场景的指导：在什么样的场景下就适合使用Kraken？

#### Webview
我想到一个重要的场景就是代替native应用中的webview，但目前还缺少与webview的性能对比。

除此之外还有哪些非常适合的使用场景呢？

**Comment by andycall** [comment]
Kraken 适合取代 hybrid web，RN，Weex 的应用场景，但是并不能完全替代 App 中的 WebView。

Kraken 是为客户端嵌入场景而设计的 Web 渲染引擎，如果你想在 App 中使用 Web 技术写一些页面，并且希望有着 Native 一样的用户体验（过场动画，动态化，打开迅速，各种客户端能力扩展），Kraken 目前所提供的功能就是一个不错的选择，因为它相比前面的技术方案更轻更快，并且可定制性更高。

如果你只是想要在 App 里面打开一个网页，并不需要一些客户端能力，仅仅是展示一个页面的话，请直接使用浏览器，而不是 Kraken。
